src/servletTutorial/validation/BookValidation.java

```diff
@@ -0,0 +1,28 @@
+package servletTutorial.validation;
+
+import java.util.List;
+
+import servletTutorial.entities.Book;
+import servletTutorial.manage.BookManagement;
+
+public class BookValidation {
+
+	public static Boolean checkEmpty(String bookName) {
+		if (bookName == "") {
+			return false;
+		}
+		return true;
+	}
+	
+	public static Boolean checkExisted(int bookId, String bookName) {
+		List<Book> listBooks = BookManagement.getBooks();
+		for(Book book: listBooks) {
+			if(book.getBookId() != bookId) {
+				if(book.getBookName().equals(bookName)) {
```

Comment:
Why do we need check `name` ?

Reply:
In the requirement, we need to verify the name.
https://docs.google.com/document/d/1AsUXeory_KFqK2CbAEJFYJPh9r983WJtR1l8lwPkAQE/edit#heading=h.jgebc6qyl86c